rubicon/objc/types.py

```diff
@@ -235,7 +238,8 @@ def ctype_for_encoding(encoding):
 
 def encoding_for_ctype(ctype):
     """Return the Objective-C type encoding for the given ctypes type."""
-    
+    if isinstance(ctype, POINTER_TYPE):
+        return b'^' + encoding_for_ctype(ctype._type_)
```

Comment:
This should probably be in the `except KeyError` block below, so that pointer types that are in `_encoding_for_ctype_map` are looked up there. For example `POINTER(c_char)` should encode as `*`, not as `^c`.